File: Week01/Even_Odd/Program.cs
using static System.Console;

class Program
{
    public static bool IsEven(int n)
    {
        int remainder = n % 2;
        if (remainder == 0)
            return true;
        else
            return false;
    }


    static void Main()
    {
        WriteLine("Enter an integer to check even or odd: ");
        int n = 0;
        n = int.Parse(ReadLine());

        if (IsEven(n))
            WriteLine("EVEN number");
        else
            WriteLine("ODD number");
    }
}

File: Week01/Even_Odd/Program.cpp
#include<iostream>

using namespace std;


bool isEven(int n) {
    int remainder = n % 2;

    if (remainder == 0) 
        return true;
    else
        return false;
}


int main() {
    cout << "Enter an integer to check even odd: ";
    int num;
    cin >> num;
    
    if (isEven(num))
        cout << "EVEN number";
    else 
        cout << "ODD number";
}
